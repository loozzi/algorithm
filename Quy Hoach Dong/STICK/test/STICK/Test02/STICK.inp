10

47
98 84
74 88
42 63
12 12
82 88
22 98
32 90
90 35
63 53
44 98
8 74
66 41
48 11
35 10
1 37
41 34
90 66
26 57
16 42
90 4
60 97
64 61
29 16
32 63
30 17
33 56
55 37
78 51
63 24
60 45
10 62
96 25
86 60
34 13
11 57
35 52
2 16
36 28
58 41
56 6
100 80
36 97
62 87
89 33
69 64
43 9
91 67

38
57 66
10 72
46 98
40 1
94 63
55 75
24 21
10 37
80 4
2 5
92 56
53 44
89 78
99 26
1 88
51 91
21 91
48 82
23 6
11 70
57 76
46 15
17 41
34 83
30 52
57 40
70 79
21 54
71 80
98 1
55 47
50 61
47 98
52 55
51 37
66 80
12 66
64 1

43
36 11
3 38
53 69
91 87
50 70
71 45
41 15
54 27
64 83
97 45
27 55
26 85
73 18
27 80
8 88
91 100
43 92
44 32
38 53
89 17
84 65
71 18
14 35
74 31
10 21
27 35
50 80
76 89
40 94
90 51
9 27
91 4
74 59
67 94
66 4
92 44
34 2
99 3
55 53
74 10
80 2
25 8
95 44

11
43 97
38 79
46 13
54 57
99 53
27 48
74 71
72 73
58 43
49 53
9 55

42
42 90
81 59
55 12
88 55
23 76
93 8
25 89
97 87
32 6
47 19
97 16
58 94
80 9
7 32
76 81
57 30
46 24
69 2
48 19
35 78
28 70
6 42
76 28
42 28
14 33
84 35
69 80
26 30
52 15
80 9
21 6
35 18
74 74
24 66
74 48
79 41
12 43
77 54
73 70
71 90
36 58
21 64

11
1 33
21 10
75 37
41 59
30 9
57 90
37 66
62 19
57 29
93 67
87 10

2
75 21
99 91

29
59 46
8 50
2 80
91 77
19 64
95 77
38 52
79 77
9 90
23 1
59 29
74 58
28 40
75 77
23 33
92 36
85 8
81 43
1 13
74 40
34 17
68 38
32 75
55 86
56 79
24 16
36 59
86 5
14 13

11
83 9
65 71
53 51
47 75
14 44
91 67
88 1
44 44
41 47
93 13
74 63

47
19 66
99 90
95 3
47 33
40 71
78 98
57 78
32 69
74 79
74 73
96 21
6 32
47 53
86 48
59 64
3 70
73 87
96 100
18 22
1 52
66 60
77 14
49 2
51 68
92 35
97 2
7 69
60 34
79 29
47 90
49 77
50 6
24 76
28 11
64 43
19 94
98 89
56 29
87 93
54 8
67 38
27 43
96 10
70 43
12 56
49 13
64 3
