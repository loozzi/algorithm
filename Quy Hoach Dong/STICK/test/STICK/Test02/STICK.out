10
10
11
4
10
5
1
8
5
10
